pkg/display/display.go

```diff
@@ -203,6 +267,16 @@ func PrintLargestDirectories(files []models.FileInformation, totalBlobs int, tot
 		100.0,
 		utils.FormatSize(totalBlobsCompressedSize),
 		100.0)
+
+	// Add footnote explaining the asterisk meaning
+	if showFootnote {
+		defaultBranch, _ := git.GetDefaultBranch()
+		if defaultBranch == "" {
+			defaultBranch = "default"
+		}
+		fmt.Println()
+		fmt.Printf("* File or directory not present in %s branch (moved or removed)\n", defaultBranch)
```

Comment:
I was thinking for a moment to make this line more "verbose", e.g.:
`* File or directory is not present in the most recent commit (abc123) of the default branch (%s) as it has been moved, renamed or removed`

Though that would make it a kind of long (wider than the tables for example).

So maybe only a small enhancement suggestion would be adding "renamed"?
```suggestion
		fmt.Printf("* File or directory not present in %s branch (moved, renamed or removed)\n", defaultBranch)
```

Maybe also adding `in latest commit`:
```suggestion
		fmt.Printf("* File or directory not present in latest commit of %s branch (moved, renamed or removed)\n", defaultBranch)
```

That last version would still be less wide than the table.

Reply:
Resolved in a69e4c67df76a4deefba822353bf7474c5bde478.